From: Didier Raboud <odyx@debian.org>
Date: Tue, 4 Oct 2016 11:44:04 +0200
Subject: Build manpages reproducibly, through reading the MODTIME from

 debian/changelog exclusively, and giving LC_ALL=C.UTF-8 TZ=UTC to various
 commands
---
 Makefile | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index ec3c3b6..6d0177e 100644
--- a/Makefile
+++ b/Makefile
@@ -80,6 +80,8 @@ endif
 ifeq ($(UNAME),SunOS)
     MODTIME= `ls -e $$1 | cut -c42-61`
 endif
+# Define modtime from the debian changelog, for all files
+MODTIME= LC_ALL=C.UTF-8 TZ=UTC date -d "$$(dpkg-parsechangelog -SDate)" "+%a %b %d %T %Y"
 
 #
 # Files for tarball
@@ -1606,12 +1608,12 @@ install-doc: doc
 	$(INSTALL) -c -m 644 ChangeLog $(DOCDIR)
 
 GROFF=/usr/local/test/bin/groff
-GROFF=groff
+GROFF=LC_ALL=C.UTF-8 TZ=UTC groff
 manual.pdf: $(MANPAGES) osx-hotplug/osx-hplj-hotplug.1
 	-$(GROFF) -t -man \
 	    `ls $(MANPAGES) \
 		osx-hotplug/osx-hplj-hotplug.1 \
-		| sort` \
+		| LC_ALL=C.UTF-8 sort` \
 	    | ps2pdf - $@
 
 README: README.in
